scripts/psra_3.Create_psra_building_all_indicators.sql

```diff
@@ -99,12 +99,17 @@ CASE WHEN ((k.structural_complete_r1/a.number) * f.collapse_pc) >0.01 THEN a.num
 
 -- 2.3 Affected People
 -- 2.3.1 Life Safety - b0
-CAST(CAST(ROUND(CAST((occupants_b0/0.000001) AS NUMERIC),6) AS FLOAT) AS NUMERIC) AS "eC_Fatality_b0",
```

Comment:
This will significantly change the indicator values, that is intended, correct?

Reply:
Yes. Previous case was intended to show the fatalities in 'micro mortalities' but after review has been decided to revert back to normal.